auslib/web/admin/swagger/api.yaml

```diff
@@ -787,6 +787,8 @@ paths:
           description: successfully created a new release
         '400':
           $ref: '#/responses/invalidFormData'
+        '401':
```

Comment:
It seems like we're missing 401s (and 403s for that matter) from a whole bunch of endpoints. Eg:   /releases/{release}/read_only,   /releases/{release}/revisions,   /users/{username}/roles/{role}, and many others - would you mind going through and adding them? No need to add new unit tests for every single endpoint.

Reply:
Ah, you are right! Indeed I added the new status in endpoints that already had 403.